lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -120,8 +121,8 @@ static void addStringOption(std::vector<std::string> &options,
 }
 
 OpenCLFunction::OpenCLFunction(std::unique_ptr<IRFunction> F,
-                               const Context &ctx)
-    : F_(std::move(F)) {
+                               runtime::RuntimeBundle bundle)
+    : F_(std::move(F)), bundle_(std::move(bundle)) {
```

Comment:
RuntimeBundle does not have efficient move copy ctor, I'd pass by const ref here as well.